courses/fundamentals_of_ada/180_polymorphism.rst

```diff
@@ -98,12 +98,10 @@ Testing the type of an object
    TC2 : T'Class := D_Obj;  -- TC2'Tag = D'Tag
```

Comment:
The naming is hard to follow, using Root and Child may be easier ?

Reply:
Done (I prefer parent and child :)